src/ILCompiler.Compiler/src/Compiler/CompilerTypeSystemContext.TypeInit.cs

```diff
@@ -15,6 +15,13 @@ namespace ILCompiler
     // Manages policies around static constructors (.cctors) and static data initialization.
     partial class CompilerTypeSystemContext
     {
+        private ExplicitClassConstructors _explicitClassConstructors;
```

Comment:
> adding CoreRT init cruft into the class library feels dirtier 

I assume that we would do a matching follow up change in the ProjectN toolchain as well, and remove the CoreRT ifdefs. So it would be same accross the board at the end.